bundles/org.openhab.core.config.core/src/test/java/org/openhab/core/config/core/internal/validation/ConfigDescriptionValidatorTest.java

```diff
@@ -241,7 +245,20 @@ public void assertValidationThrowsExceptionContainingMessagesForAllRequiredConfi
         }
     }
 
-    void assertRequired(String parameterName) {
+    void assertMissingRequired(String parameterName) {
+        List<ConfigValidationMessage> expected = List.of(new ConfigValidationMessage(parameterName,
+                MessageKey.PARAMETER_REQUIRED.defaultMessage, MessageKey.PARAMETER_REQUIRED.key));
+
+        try {
+            params.remove(parameterName);
+            configDescriptionValidator.validate(params, CONFIG_DESCRIPTION_URI);
+            failBecauseOfMissingConfigValidationException();
+        } catch (ConfigValidationException e) {
+            assertThat(getConfigValidationMessages(e), is(expected));
+        }
```

Comment:
JUnit supports checking expected exceptions with `Assertions.assertThrows` like this nowadays:

```suggestion
        params.remove(parameterName);
        ConfigValidationException exception = assertThrows(ConfigValidationException.class,
                () -> configDescriptionValidator.validate(params, CONFIG_DESCRIPTION_URI));
        assertThat(getConfigValidationMessages(exception), is(expected));
```

Reply:
Done. Also for all other occurrences in this file.